src/components/task-plan/reading.cjsx

```diff
@@ -38,7 +40,22 @@ SelectTopics = React.createClass
     return
 
   render: ->
-    <BS.Button onClick={@handleToggle} bsStyle="primary">Edit Readings</BS.Button>
+    if (@props.selected.length and TocStore.isLoaded())
```

Comment:
There should probably be an `else` with `selectedReadingList = <div className="loading">Loading...</div>`

Also, (minor) what if `TocActions.load()` errors?